workmail-save-and-update-email/tst/env_vars.json

```diff
@@ -0,0 +1,13 @@
+{
+  "WorkMailSaveAndUpdateEmailFunction": {
+      "SAVED_EMAIL_BUCKET": "SAVED_EMAIL_BUCKET",
+      "UPDATED_EMAIL_BUCKET": "UPDATED_EMAIL_BUCKET",
+      "DISCLAIMER": "YOUR_CUSTOM_INTERNAL_DISCLAIMER_TEXT {key}",
```

Comment:
Remove `{key}` part?

Reply:
I was hoping to show an example of how to template it in. Maybe there is a better way.